### PR TITLE
Added Validations KHADRI-09112024-0716-PM

### PR DESCRIPTION
**Pull request by roohishaik128**
Added Validations

### CODE REVIEW

File: src/main/java/com/khadri/jakarta/product/servlet/delete/ProductDeleteSearchPageServlet.java [hunk 1]
```diff
@@ -35,16 +35,26 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 
 		sb.append("<html>");
 		sb.append("<body>");
+		sb.append("<script type='text/javascript'>");
+		sb.append("function formValidation() {");
+		sb.append("var productIdComponent = document.getElementById('productId');");
+		sb.append("if (productIdComponent.value == '') {");
+		sb.append("alert('Please Enter Product Id.');");
```
Comment (SalukulaMahendra) [inline src/main/java/com/khadri/jakarta/product/servlet/delete/ProductDeleteSearchPageServlet.java:42]: remove . in line number 42

File: src/main/java/com/khadri/jakarta/stock/servlet/view/StockViewServlet.java [hunk 2]
```diff
@@ -108,7 +130,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 			case "mobile":
 				listOfMobileForm = stockDao.viewMobileData(product_brand, product_model);
 				if (listOfMobileForm != null && !listOfMobileForm.isEmpty()) {
-					 
+
```
Comment (SalukulaMahendra) [inline src/main/java/com/khadri/jakarta/stock/servlet/view/StockViewServlet.java:133]: remove space

File: src/main/webapp/add_product.html [hunk 3]
```diff
@@ -3,9 +3,22 @@
 <html>
 <head>
 <link rel='stylesheet' type='text/css' href='styles.css' />
+<script type="text/javascript">
+	function formValidation() {
+		var productNameComponent = document.getElementById("productName");
+
+		if (productNameComponent.value == "") {
+			;
+			alert("Please Enter ProductName ..!");
```
Comment (SalukulaMahendra) [inline src/main/webapp/add_product.html:12]: remove .and !

**Review by SalukulaMahendra — COMMENTED** [review]
resolve review comments

File: src/main/java/com/khadri/jakarta/product/servlet/delete/ProductDeleteSearchPageServlet.java (hunk 1 above)
Comment (roohishaik128) [reply src/main/java/com/khadri/jakarta/product/servlet/delete/ProductDeleteSearchPageServlet.java:42]: I Resolved

File: src/main/webapp/add_product.html (hunk 3 above)
Comment (roohishaik128) [reply src/main/webapp/add_product.html:12]: I Resolved

File: src/main/java/com/khadri/jakarta/stock/servlet/view/StockViewServlet.java (hunk 2 above)
Comment (roohishaik128) [reply src/main/java/com/khadri/jakarta/stock/servlet/view/StockViewServlet.java:133]: I Resolved

**Review by manvithagali52 — APPROVED** [review]

**Review by thasmiyas39 — APPROVED** [review]

**Review by khadri2021 — APPROVED** [review]

**Pull request merged**